### PR TITLE
docker: uwsgi usage instead of gunicorn

### PR DESCRIPTION
**Pull request by hachreak**
Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

**Comment by krzysztof** [comment]
Hey,

``requirements.developer.txt`` is no longer in the repo: https://github.com/zenodo/zenodo/pull/652/files#diff-3254677a7917c6c01f55212f86c57fbfR44

use ``requirements.txt`` instead.

**Comment by hachreak** [comment]
I see... I'm trying to fix it :)

**Comment by hachreak** [comment]
ping @lnielsen

File: uwsgi.ini [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[uwsgi]
```
Comment (lnielsen) [inline uwsgi.ini]: Please move this file to ``docker/uwsgi/``

File: uwsgi.ini [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+http = 0.0.0.0:5000
+wsgi-file = zenodo/wsgi.py
```
Comment (lnielsen) [inline uwsgi.ini]: please use ``module`` - http://uwsgi-docs.readthedocs.io/en/latest/Options.html#module

File: zenodo/wsgi.py [hunk 3]
```diff
@@ -26,7 +26,10 @@
 
 from __future__ import absolute_import, print_function
 
-from .factory import create_app
+from zenodo.factory import create_app
```
Comment (lnielsen) [inline zenodo/wsgi.py]: all changes here are not necessary

File: uwsgi.ini (hunk 2 above)
Comment (hachreak) [inline uwsgi.ini]: what should I do for virtualenv?

**Pull request closed without merging**